hw/top_earlgrey/BUILD

```diff
@@ -92,8 +92,14 @@ fpga_cw310(
     base = ":fpga_cw310_test_rom",
     exec_env = "fpga_cw310_rom_with_fake_keys",
     manifest = "//sw/device/silicon_creator/rom_ext:manifest_standard",
-    rom = "//sw/device/silicon_creator/rom:rom_with_fake_keys",
-    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:test_private_key_0": "test_key_0"},
+    rom = select({
+        "//signing:test_keys": "//sw/device/silicon_creator/rom:rom_with_fake_keys",
```

Comment:
nit: Does this mean that this target can support either fake or real keys? If so, we should consider updating this to s/`fpga_cw310_rom_with_fake_keys`/`fpga_cw310_rom`/ in a future PR.